roles/ceph-osd/tasks/scenarios/raw_multi_journal.yml

```diff
@@ -26,4 +27,13 @@
     - raw_multi_journal
     - not osd_auto_discovery
 
+- name: fail if ceph-disk cannot prepare an OSD
+  fail:
+    msg: "ceph-disk failed to prepare an OSD"
```

Comment:
I don't think we can reliably suggest something because you can get numerous failures. Just like what we do for activating OSDs this task fails here immediately after the previous error.